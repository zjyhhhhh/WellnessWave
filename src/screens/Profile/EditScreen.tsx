import { View, Text } from "react-native";

const EditScreen = ({}) => {
	return (
		<View>
			<Text>Edit Screen</Text>
		</View>
	);
};

export default EditScreen;
