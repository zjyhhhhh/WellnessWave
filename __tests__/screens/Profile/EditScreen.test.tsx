import "@testing-library/jest-dom";
import renderer from "react-test-renderer";
import EditScreen from "../../../src/screens/Profile/EditScreen";
import { render } from "@testing-library/react-native";

describe("Edit Screen", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<EditScreen />).toJSON();
		expect(tree).toMatchSnapshot();
	});

	it("displays the 'Edit Screen' text", () => {
		const { getByText } = render(<EditScreen />);
		expect(getByText("Edit Screen")).toBeTruthy();
	});
});
