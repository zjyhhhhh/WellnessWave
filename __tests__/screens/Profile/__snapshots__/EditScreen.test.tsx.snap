// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit Screen renders correctly 1`] = `
<View>
  <Text>
    Edit Screen
  </Text>
</View>
`;
